draft-ietf-httpbis-message-signatures.md

```diff
@@ -341,6 +341,30 @@ The following example shows canonicalized values for different component identif
 
 Note that the value for `key="c"` has been re-serialized.
 
+### Absent HTTP Fields
+
+If the signer wants to signal that an HTTP field is not present in the signed message at the time the signature is generated, the signer can include that field in the signature with the parameter `absent` to indicate that the field MUST NOT be present in the message.
+
+The component value is the empty string.
+
+For example, for the following message:
+
+~~~ http-message
+HTTP/1.1 201 No Content
+Date: Wed, 16 Feb 2022 04:20:23 GMT
+~~~
+
+The signer wishes to indicate that a `Content-Encoding` header is not included in the response being signed, which can be shown as presented using the signature input string format discussed in {{create-sig-input}}:
+
+~~~
+NOTE: '\' line wrapping per RFC 8792
+
+"content-encoding";absent: \
```

Comment:
The use of '\' line wrapping confused me here.  It reads line you aren't terminating the line, but I think that the blank line that follows implies that the line is terminated.  The note clarifies that the terminator is to capture the trailing space, but that seems unnecessary (see above).

Reply:
We did something similar with the "empty header" examples for the same reasons. The tools strip off the trailing space from examples, which messed up people trying to build the examples correctly. Happy to hear other solutions, but this does give the right output when the line unwrapping is applied.

Reply:
I take it the whitespace is required by the canonical format you've chosen? If so, what about removing it?